docs/qcqmc/full_workflow.ipynb

```diff
@@ -0,0 +1,363 @@
+    "    do_pp=True,\n",
+    "    restricted=False,\n",
+    "    initial_orbital_rotation=None,\n",
+    "    initial_two_body_qchem_amplitudes=np.asarray([0.3, 0.4]),\n",
+    "    do_optimization=True,\n",
+    "    use_fast_gradients=True\n",
+    ")\n",
+    "\n",
+    "trial_wf = TrialWavefunctionData.build_pp_plus_trial_from_dependencies(\n",
+    "    pp_params, dependencies={pyscf_params: pyscf_hamiltonian}, do_print=True\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(f\"Trial wavefunction energy: {trial_wf.ansatz_energy}\")\n",
+    "print(f\"Exact ground state energy: {trial_wf.fci_energy}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from recirq.qcqmc.blueprint import BlueprintParamsTrialWf, BlueprintData\n",
+    "from recirq.qcqmc.experiment import SimulatedExperimentParams, ExperimentData\n",
+    "\n",
+    "blueprint_params = BlueprintParamsTrialWf(\n",
+    "    name=\"blueprint_test_medium\",\n",
+    "    trial_wf_params=pp_params,\n",
+    "    n_cliffords=100,\n",
+    "    qubit_partition=(tuple(qubit for qubit in pp_params.qubits_jordan_wigner_ordered),),\n",
+    "    seed=1,\n",
+    ")\n",
+    "\n",
+    "blueprint = BlueprintData.build_blueprint_from_dependencies(blueprint_params, dependencies={pp_params: trial_wf})\n",
+    "\n",
+    "simulated_experiment_params = SimulatedExperimentParams(\n",
+    "    name=\"test_1\",\n",
+    "    blueprint_params=blueprint.params,\n",
+    "    noise_model_name=\"None\",\n",
+    "    noise_model_params=(0,),\n",
+    "    n_samples_per_clifford=31,\n",
+    "    seed=1,\n",
+    ")\n",
+    "experiment = ExperimentData.build_experiment_from_dependencies(\n",
+    "    params=simulated_experiment_params, dependencies={blueprint.params: blueprint}\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from typing import Dict\n",
+    "from recirq.qcqmc.data import Params, Data\n",
+    "from recirq.qcqmc.analysis import OverlapAnalysisData, OverlapAnalysisParams\n",
+    "\n",
+    "analysis_params = OverlapAnalysisParams(\n",
+    "    \"TEST_analysis\", experiment_params=experiment.params, k_to_calculate=(1,)\n",
+    ")\n",
+    "all_dependencies: Dict[Params, Data] = {\n",
+    "    pyscf_params: pyscf_hamiltonian,\n",
+    "    pp_params: trial_wf,\n",
+    "    blueprint_params: blueprint,\n",
+    "    simulated_experiment_params: experiment,\n",
+    "}\n",
+    "analysis = OverlapAnalysisData.build_analysis_from_dependencies(\n",
+    "    analysis_params, dependencies=all_dependencies\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally we save the wavefunction in a format that is acceptable for ipie."
```

Comment:
nit: I think there should be a comma after "Finally"

Reply:
Done.